### PR TITLE
Refactored some param parsing in main

### PR DESCRIPTION
**Pull request by loganintech**
Was reading through the repo the other day and felt I could contribute a bit. Let me know if you want any info or explanations.

### CODE REVIEW

**Comment by loganintech** [comment]
@rbradford Of course! Was this more what you were looking for?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ for each submitted Pull Request (PR).
 
 ## Certificate of Origin
 
-In order to get a clear contribution chain of trust we use the [signed-off-by language] (https://01.org/community/signed-process)
+In order to get a clear contribution chain of trust we use the [signed-off-by language](https://01.org/community/signed-process)
```
Comment (rbradford) [inline CONTRIBUTING.md]: Good catch!

**Comment by sboeuf** [comment]
@loganintech @rbradford We need to fix this patch since it breaks the `--net` behavior. Using an empty `--net` does not give us any network, which is not the expected behavior here.

**Comment by loganintech** [comment]
Shoot, I can fix this later today if it doesn't get fixed by the time I get to it @sboeuf

**Comment by loganintech** [comment]
Although... Is it even possible to call net without a parameter? Looking back the reason I left it like this is because clap requires an argument is passed to net and even if that's an empty string it's still filtered through to_string.

**Pull request merged**